src/components/Home.module.css

```diff
@@ -0,0 +1,183 @@
+.home_cards{
```

Comment:
Here you can create `Common.module.css` and decouple and reduce few classes and things from this `Home.module.css` and you can create `Post.module.css` that will be used specifically for styling the posts page/pages.